.gitignore

```diff
@@ -0,0 +1,2 @@
+# Chart dependencies refs https://github.com/helm/charts/blob/master/.gitignore
+**/charts/*.tgz
```

Comment:
When we run `helm dependency build`, 'envoy-${VERSION}.tgz' will be `downloaded` to this directory, so we can exclude it from git management